### PR TITLE
support lists in field_value

### PR DESCRIPTION
**Pull request by dsego**
@martynjames 
Not sure if this is the right way to do it, but it fixes the bokchoy tests for dashboard search. I did not write a unit test for this change yet.

### CODE REVIEW

**Comment by martynjames** [comment]
Looks like the right thing to do - please add a test to check both ways, and run it for both Mock and Elastic implementations.

**Comment by edx-webhook** [comment]
Thanks for the pull request, @dsego! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request. 

To automatically create an OSPR issue for this pull request, just visit this link: http://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fedx-search&number=9

**Comment by martynjames** [comment]
:+1:

**Comment by dsego** [comment]
@dino-cikatic can you do a review as well?

File: search/elastic.py [hunk 1]
```diff
@@ -45,6 +46,11 @@ def translate_result(result):
     return response
 
 
+def _is_iterable(item):
```
Comment (dino-cikatic) [inline search/elastic.py]: Why are we coping the same function twice? Maybe you should move this over to helpers.

**Comment by dino-cikatic** [comment]
Once that one small note is resolved, it looks good to me :+1:

**Comment by dsego** [comment]
@martynjames  Does the last commit look ok?

**Comment by martynjames** [comment]
@dsego - thanks a lot! Please squash and then merge. :+1:

**Pull request merged**